server/neptune/workflows/internal/deploy/workflow.go

```diff
@@ -49,11 +48,6 @@ func Workflow(ctx workflow.Context, request Request, terraformWorkflow func(ctx
 	}
 	ctx = workflow.WithActivityOptions(ctx, options)
 
-	// set relevant logging context
-	ctx = workflow.WithValue(ctx, config.RepositoryLogKey, request.Repository.FullName)
-	ctx = workflow.WithValue(ctx, config.ProjectLogKey, request.Root.Name)
-	ctx = workflow.WithValue(ctx, config.GHRequestIDLogKey, request.GHRequestID)
```

Comment:
why not continue to add these fields to the context (using `internalContext`)?

Reply:
The context propagator should add it based on what's passed in.